packages/frontend/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -454,24 +454,7 @@ async function onWorkflowMenuSelect(action: WORKFLOW_MENU_ACTIONS): Promise<void
 			break;
 		}
 		case WORKFLOW_MENU_ACTIONS.IMPORT_FROM_URL: {
-			try {
-				const promptResponse = await message.prompt(
-					locale.baseText('mainSidebar.prompt.workflowUrl') + ':',
-					locale.baseText('mainSidebar.prompt.importWorkflowFromUrl') + ':',
-					{
-						confirmButtonText: locale.baseText('mainSidebar.prompt.import'),
-						cancelButtonText: locale.baseText('mainSidebar.prompt.cancel'),
-						inputErrorMessage: locale.baseText('mainSidebar.prompt.invalidUrl'),
-						inputPattern: VALID_WORKFLOW_IMPORT_URL_REGEX,
-					},
-				);
-
-				if (promptResponse.action === 'cancel') {
-					return;
-				}
-
-				nodeViewEventBus.emit('importWorkflowUrl', { url: promptResponse.value });
-			} catch (e) {}
+			uiStore.openModal('IMPORT_WORKFLOW_URL_MODAL_KEY');
```

Comment:
This should not be in quotes:
```suggestion
			uiStore.openModal(IMPORT_WORKFLOW_URL_MODAL_KEY);
```